pennylane_lightning/lightning_qubit.py

```diff
@@ -160,6 +160,12 @@ class LightningQubit(QubitDevice):
             the expectation values. Defaults to ``None`` if not specified. Setting
             to ``None`` results in computing statistics like expectation values and
             variances analytically.
+        mcmc (bool): Determine whether we use Markov Chain Monte Carlo sampling method when generating samples.
```

Comment:
```suggestion
        mcmc (bool): Determine whether to use the approximate Markov Chain Monte Carlo sampling method when generating samples.
```
I want to emphasize that this is an approximate method.